integration-tests/hibernate-orm-panache/src/main/java/io/quarkus/it/panache/TestEndpoint.java

```diff
@@ -789,4 +789,15 @@ public Person ignoredProperties() throws NoSuchMethodException, SecurityExceptio
         person.status = Status.DECEASED;
         return person;
     }
+
+    @Inject
+    Bug5274EntityRepository bug5274EntityRepository;
+
+    @GET
+    @Path("5274")
+    @Transactional
+    public String testBug5274() {
```

Comment:
Maybe named it testWithAbstractParentEntity and add a link to the bug will be more explainatory ?

Reply:
I prefer having the issue number in the test name because that really has more info than whatever name I could come up with. We could add a link as comment, but I've never felt the need for it.

Reply:
OK, as you prefere :)